routes/index.js

```diff
@@ -4,10 +4,22 @@ var router = express.Router();
 // Require controller.js:
 var controller = require('../controllers/controller.js');
 
-/* GET home page. */
+// GET home page:
 router.get('/', controller.homePage);
 
-// GET detail page:
-router.get('/detail/:id', controller.detailPage);
+// GET new story page:
+router.get('/new-story', controller.newStoryPage);
+
```

Comment:
Add: `router.post('/new-story', controller.newStoryPage);`

Redirect to /create-story

Reply:
No I wont :P